spec/models/user_spec.rb

```diff
@@ -25,6 +25,12 @@
 
   it { is_expected.to be_valid }
 
+  context "when user has email address with apostrophe" do
+    subject(:user) { create(:user, email: "o'niel@example.com") }
```

Comment:
I see it's like this in other tests, but this could probably be `build`, and then we wouldn't need to hit the database for this test.